venia-integration-tests/src/tests/integrationTests/customerService/customerService.spec.js

```diff
@@ -0,0 +1,22 @@
+import { graphqlMockedCalls as graphqlMockedCallsFixtures } from '../../../fixtures';
+import { clickOnfooterLinks } from '../../../actions/customerPage';
```

Comment:
@itsyanik Test is failing Please change this to `import { clickOnfooterLinks } from '../../../actions/footer';`

Even after updating above it still fails with error, please check - 
```
 1) Customer Service Page
       Verify content:
     CypressError: Timed out retrying after 30000ms: `cy.wait()` timed out waiting `30000ms` for the 1st request to the route: `getCMSMockData`. No request ever occurred.
```

Reply:
Error was because of call never happened as page was not existed in backend. Created dummy page until we update backend with sample data meta package.  Snap update was need as snap needs to be saved via `yarn test:headless` and not via `yarn test` cypress UI.